.github/workflows/test.yml

```diff
@@ -25,14 +25,27 @@ jobs:
     - name: Install pip
       run: apt update && apt install -y python3-pip python3-pytest
 
+    - name: Install apt dependencies
+      run: apt update && apt install -y doxygen graphviz
+
+    - name: Smoke test of pipx install
```

Comment:
Can we drop this into a separate test to keep it clear that it's a separable test. It will run in parallel and get a separate report then too.